etc/DAQ_VERSION

```diff
@@ -1 +1 @@
-1.9.13
+master
```

Comment:
Is there a specific DAQ version/change you need?  It's trivial for us now to tag an intermediate DAQ version (e.g. 1.9.14) -- so we can just do that if necessary.

Reply:
Using 1.9.14 now